css/index.css

```diff
@@ -1,11 +1,36 @@
-.row {
-    background-color:  rgba(251, 147, 0, .5);
-    }
+.container{
+    height: 400px;
+    width:600;
+    display: flex;
+    justify-content: space-evenly;
+    background-color: backdrop #F5E6CA
+}
+
+.textHere{
+    text-align: left;
+}
+
+.person{
+    width: 250px;
+    height: 500px;
+    display: flex;
+    justify-content: right;
+}
     
-    .cards{ 
-    width: 150x;
-    height: 300px;
+.nameHere{
+    width: 250px;
+    height: 500px;
+    display: flex;
+    justify-content: left;
+}
+
+
+
+.cards{ 
+    width: 100x;
+    height: 200px;
     border: solid 5px #FB9300; 
```

Comment:
Sorry Mary, this is the right color we selected! I had them listed as RGB values but you listed a hex color. This color works.

Reply:
ok cool